src/brpc/trackme.cpp

```diff
@@ -43,11 +43,11 @@ static const int32_t TRACKME_MIN_INTERVAL = 30;
 static const int32_t TRACKME_MAX_INTERVAL = 600;
 static int32_t s_trackme_interval = TRACKME_MIN_INTERVAL;
```

Comment:
s_traceme_interval => g_traceme_interval?

Reply:
"s" stands for "static"